src/Admin/BaseFieldDescription.php

```diff
@@ -132,6 +132,25 @@ abstract class BaseFieldDescription implements FieldDescriptionInterface
      */
     private static $fieldGetters = [];
 
+    /**
+     * NEXT_MAJOR: Remove the null default value and restrict param type to `string`.
+     */
+    public function __construct(?string $name = null, array $options = [])
+    {
+        // NEXT_MAJOR: Remove this check and keep the else part.
+        if (null === $name) {
```

Comment:
The value being `null` is not the same as "passing no argument".
```suggestion
        if (null === $name && 0 === \func_num_args()) {
```

Reply:
@phansys null will be a forbidden value too. Only string will be allowed in next major

Reply:
Oh, good. Thanks.
Then we need to update the deprecation message.